doc/access.md

```diff
@@ -22,6 +22,27 @@ of members is [here][Build WG Members].
 A subsection of build members have access to infra machines
 (`infra-*`). The list of members is [here][Infra Admins].
 
+The infra group also have access to:
+
+#### Servers:
+- [DigitalOcean Droplets][] (individual accounts)
+- [Joyent][]
+- [MacStadium][]
+- [Packet.net][] (individual accounts)
+- [Rackspace][]
```

Comment:
we should make this individual accounts, this is possible now and I have one

Reply:
https://github.com/nodejs/build/issues/1072